src/entries/popup/pages/messages/AddEthereumChain/index.tsx

```diff
@@ -79,6 +78,13 @@ export const AddEthereumChain = ({
         dappDomain: dappMetadata?.appHost || '',
         dappName: dappMetadata?.appName,
       });
+
+      // wait 1.2s for storages to sync (persist throttle on network store is 1s)
+      await new Promise((resolve) => {
+        setTimeout(resolve, 1200);
+      });
+
+      approveRequest(true);
```

Comment:
Would keep this above the analytics event in case that is ever blocking

Reply:
I updated the analytics to make it non-blocking

I think keeping it infront of the timeout is a good idea as it gives more time for the network request to succeed
approveRequest() will close the popup and the event may never gets delivered